cmd/embedded-cluster/reset.go

```diff
@@ -299,7 +299,7 @@ var resetCommand = &cli.Command{
 		},
 		&cli.BoolFlag{
 			Name:  "force",
-			Usage: "Ignore errors encountered when resetting the node. Implies --no-prompt.",
+			Usage: "Ignore errors encountered when resetting the node. Implies --no-prompt",
```

Comment:
If we're going to remove periods, let's do something like this
```suggestion
			Usage: "Ignore errors encountered when resetting the node (implies --no-prompt)",
```